py-polars/tests/unit/operations/rolling/test_map.py

```diff
@@ -86,7 +93,7 @@ def test_rolling_map_sum_int_cast_to_float() -> None:
         function=lambda s: s.sum(), window_size=3, weights=[1.0, 2.0, 3.0]
     )
 
-    expected = pl.Series("A", [None, None, 32.0, 20.0, 48.0], dtype=pl.Float64)
+    expected = pl.Series("A", [None, None, 32.0, None, None], dtype=pl.Float64)
```

Comment:
This test was previously incorrect, as it did not take into account the null values in the window.